### PR TITLE
CM-31107 - Fix new CLI endpoints that should be applied to SCA scans only

### PR DESCRIPTION
**Pull request by MarshalX**

### CODE REVIEW

**Comment by MarshalX** [comment]
@morsa4406 pls review

**Review by artem-fedorov — DISMISSED** [review]

File: cycode/cyclient/scan_client.py [hunk 1]
```diff
@@ -20,22 +20,48 @@ def __init__(
...
+
+        return self._DETECTIONS_SERVICE_CONTROLLER_PATH
+
+    def get_scan_service_url_path(self, scan_type: str) -> str:
+        service_path = self.scan_config.get_service_name(scan_type)
+        controller_path = self.get_scan_controller_path(scan_type)
+        return f'{service_path}/{controller_path}'
+
+    def get_scan_url_path(self, scan_type: str) -> str:
+        prefix_path = self.scan_config.get_scans_prefix()
```
[the remark this answers is not in the document]
Comment (MarshalX) [reply cycode/cyclient/scan_client.py:51]: the first one depends on scan type and the second one is not. idk what the difference is and why the get_scans_prefix even exists in the configs xd. maybe @MichalBor remember

[the remark this answers is not in the document]
Comment (MarshalX) [reply cycode/cyclient/scan_client.py:51]: i double-check. looks like the initial code written by you ;)

[the remark this answers is not in the document]
Comment (morsa4406) [reply cycode/cyclient/scan_client.py:51]: Ok @MichalBor wdyt? there is a reason that there is two different build scan url?

[the remark this answers is not in the document]
Comment (MarshalX) [reply cycode/cyclient/scan_client.py:51]: probably we can delete get_scans_prefix from configs at all? and use get_service_name

[the remark this answers is not in the document]
Comment (MarshalX) [reply cycode/cyclient/scan_client.py]: removed

**Review by morsa4406 — APPROVED** [review]

**Pull request merged**